src/core/runner.ts

```diff
@@ -237,7 +237,11 @@ export default class Runner {
       const page = pages[pages.length - 1];
       if (page) {
         data.url ??= page.url();
-        if (screenshots && screenshots !== 'off') {
+        if (
+          screenshots &&
+          screenshots !== 'off' &&
+          data.url !== 'about:blank'
```

Comment:
a few questions:

- Why did you add `about:blank`? Have you reproduced any bugs?
- Although I'm assuming that you don't want to screenshot "about:blank" pages.
-  Does it make sense to expect customers to have any scenario where they want to open **about:blank** for any reason? In that case, maybe keep doing the screenshot it might make sense since that is their choice. I understand that seems a weird scenario, I'm just sharing a thought.

Either if the "about:blank" is because of a bug or because of the sake of efficiency/performance improvement, I would add a comment saying the why. Otherwise, it might be a source of questions in the future (for us and for future contributors) 

A test could also be a good way to document this

Reply:
I read again your PR's description, perhaps I would add the comment adding "why" based on that comment.